_episodes/03-monitor-the-model.md

```diff
@@ -514,31 +481,29 @@ from tensorflow.keras.layers import BatchNormalization
 ~~~
```

Comment:
This needs to be formatted as an exercise, and a bit of introduction, such as:
```suggestion
Look at the [documentation of the batch normalization layer](https://keras.io/api/layers/normalization_layers/batch_normalization/). Add this as a first layer to the model we defined above. Train the model and compare the performance to the model without batch normalization.
~~~
```

Reply:
Sounds good to me. Thanks for adding this!